packages/mesos/README.md

```diff
@@ -1,7 +1,8 @@
 <H2>Patches to cherry-pick on top of open source Apache Mesos before building in DC/OS</h2>
 These commits can be found in the repository at <a href="https://github.com/mesosphere/mesos/">https://github.com/mesosphere/mesos/</a>:
-<li>[42e4e19d0d7e54d5e361e1dd2d6afbd24d1928d1] Set LIBPROCESS_IP into docker container.
-<li>[09975e642168a47c466e211f02a590b6e3e1e40c] Changed agent_host to expect a relative path.
-<li>[d2cf9ac1dfa0d7d2b5ebcf8f9da86190706bb5cd] Mesos UI: Change paths, pailer, and logs to work with a reverse proxy.
-<li>[3d042bae0431da237d2c109525ea81fe8eb9943c] Revert "Fixed the broken metrics information of master in WebUI."
-<li>[dc01a78680410d7fe49e096508b64e94d991471c] Updated mesos containerizer to ignore GPU isolator creation failure.
+<li>[0b3764964acb8e621bf4af0e66ed29bc6f53d5e7] Set LIBPROCESS_IP into docker container.
+<li>[325ddadc308db79c0a2b57293d94abc3142c6d98] Changed agent_host to expect a relative path.
+<li>[41fd07582ac4357a5a5fe7b86b3ac7f18f179dcb] Mesos UI: Change paths, pailer, and logs to work with a reverse proxy.
+<li>[acb6ac9fbdeb3a97ccfa960d311617d73bb34d53] Revert "Fixed the broken metrics information of master in WebUI."
+<li>[27a6389c996c232acb43a4780f9c5330bca4924b] Updated mesos containerizer to ignore GPU isolator creation failure.
+<li>[6adb3a3071524b87d7ce97d371e82aa2a7af6cbd] Updated the UI to fix maintenance in DC/OS.
```

Comment:
This one-liner will have to be updated again once we bump to a more recent 1.2.x that includes https://github.com/apache/mesos/commit/bec31a51c4c4fc21fa0f08c68d91084f9a559450